app/controllers/v1/sessions_controller.rb

```diff
@@ -138,23 +138,49 @@ def user_login(saml_response)
     end
 
     def render_login(type)
-      login_url, post_params = login_params(type)
+      login_url, @post_params = login_params(type)
```

Comment:
why change this to an instance variable?

Reply:
Good catch.  That's left over from a previous refactoring that I forgot to revert.